tasks/tasks.py

```diff
@@ -275,9 +276,10 @@ def _before(self):
                     action_name=self.action_name),
                 os.path.join(self.data_dir, 'vars.yml'),
                 dict(repofile_url=urllib.parse.urljoin(
-                        self.build_url, 'rpms/freeipa-prci.repo'),
-                     update_packages=self.update_packages,
-                     selinux_enforcing=self.selinux_enforcing))
+                    self.build_url, 'rpms/freeipa-prci.repo'),
```

Comment:
On the other hand, I think we can replace this `dict()` with a beautiful literal. :D

Reply:
@netoarmando I hope this is what you meant :)

Reply:
it is, maybe a little more formatted, like this:
```python
constants.ANSIBLE_VARS_TEMPLATE.format(
    action_name=self.action_name),
    os.path.join(self.data_dir, 'vars.yml'),
    {
        "repofile_url": urllib.parse.urljoin(
            self.build_url, 'rpms/freeipa-prci.repo'
        ),
        "update_packages": self.update_packages,
        "selinux_enforcing": self.selinux_enforcing,
        "fips": self.fips,
    }
)
```